posts/2022-10-19-runtime-multistage-programming.adoc

```diff
@@ -0,0 +1,315 @@
+# Custom JIT compilation with Runtime Multi-Stage Programming
+
+For a while now I've been fiddling with a library called [Slinc](https://github.com/markehammons/slinc), a way to link C code in Scala. Since my original version, I've been using context values, macros, and other tools to generate code for writing and reading representations of Structs into and out of native memory.
```

Comment:
Sorry - used the Markdown syntax instead of the asciidoc one.
```suggestion
For a while now I've been fiddling with a library called https://github.com/markehammons/slinc[Slinc], a way to link C code in Scala. Since my original version, I've been using context values, macros, and other tools to generate code for writing and reading representations of Structs into and out of native memory.
```